metagpt/actions/run_code.py

```diff
@@ -152,11 +153,23 @@ def _install_via_subprocess(cmd, check, cwd, env):
         return subprocess.run(cmd, check=check, cwd=cwd, env=env)
 
     @staticmethod
-    def _install_dependencies(working_directory, env):
+    def _install_requirements(self, working_directory, env):
```

Comment:
@staticmethod will ignore this self. is it a bug?

Reply:
my bad, thanks for pointing out :+1: